tox.ini

```diff
@@ -7,7 +7,7 @@ envlist =
     integration
 
 [testenv]
-passenv = *
+passenv = DD_TEST_CLIENT*
```

Comment:
To avoid accidentally using things like API/APP key env vars?

Reply:
Yes. Because we invoke `dog` as a subprocess directly, and it uses the regular env variables by default